### ISSUE TITLE
New features

### ISSUE THREAD

**Issue by ffabrizio4** [nextcloud/deck#778]
Hi and congratulations for your work.
I think it would be useful to add the possibility of also vertically placing each new note. It would also be useful to add the possibility of multiple selection of the notes (check) and subtitles. 
Thank you for your work.

**Comment by juliushaertl** [comment]
Thanks for your feature request. I'm sorry, but I don't get exactly what you mean. Can you rephrase it and maybe give some example of how that should look like?

**Comment by juliushaertl** [comment]
As there is no report since a while. I will close this ticket. If this is still valid please feel free to reopen.
